docs/source/index.rst

```diff
@@ -6,12 +6,11 @@
 Pyro API
 ========
 
-The ``pyroapi`` package dynamically dispatches among multiple Pyro backends, including standard Pyro_, NumPyro_, Funsor_, and custom user-defined backends.
+The ``pyroapi`` package dynamically dispatches among multiple Pyro backends, including standard Pyro_, NumPyro_, and custom user-defined backends.
```

Comment:
Could we please keep the docs and pointers to funsor, so as to demonstrate the value of `pyroapi`?